config/default.yml

```diff
@@ -106,6 +106,12 @@ RSpec/AroundBlock:
   VersionAdded: '1.11'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/AroundBlock
 
+RSpec/AvoidSetupHook:
+  Description: Checks that tests use RSpec's `before` over rspec-rails' `setup` method.
```

Comment:
`setup` is not an `rspec-rails`', but it rather comes from Rails itself.
See https://github.com/rspec/rspec-rails/blob/ee251b792646beb239f8b614fdc13ec111885a66/lib/rspec/rails/adapters.rb#L106

There might also be a `teardown do` hook that should canonically be defined as `after do`.

Reply:
rspec-rails or rails proper, doesn't this cop belong in our `RSpec::Rails` namespace?